### DISCUSSION TITLE
[Feature] Ability to set local and remove IP address on Immich phone app

### DISCUSSION THREAD

**Discussion by T0biasCZe** [immich-app/immich#13263] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently, in the Immich app you set only 1 IP address. This causes that you can only set either local IP address, which allows you to quickly copy photos to immich instance at home, but not be able to access immich when not at home, or, set the Internet URL, which allows you to be able to access immich from everywhere, however, the connection is slow even at home, since it has to go the internet and then reverse back to the server

One solution would be to, same as like how Home Assistant works, have a input field in the mobile app for the Home Wi-Fi SSID, and then user is at home and connected to the SSID, it would connected over local IP, and if user is on mobile data or other wifi, it would connect over remote wifi

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by mmomjian** [comment]
Thank you for your message. This is a duplicate of #308 , and we have decided that we will not be adding this feature to Immich. It can be accomplished using hairpin NAT or local DNS server.
